python/sglang/srt/layers/radix_attention.py

```diff
@@ -46,6 +47,9 @@ def __init__(
         self.v_head_dim = v_head_dim if v_head_dim != -1 else head_dim
         self.scaling = scaling
         self.layer_id = layer_id
+        self.sliding_window_size = (
+            sliding_window_size - 1 if sliding_window_size != -1 else -1
```

Comment:
I am curious why is this `sliding_window_size-1`?

Reply:
To align with HF, we use the sliding window inclusively, while flashinfer uses it exclusively for the last token.